docs/source/system_admins/commandline_install.rst

```diff
@@ -0,0 +1,62 @@
+.. _command_line_install:
+
+Command line install
```

Comment:
Should this be `Command Line Install` ? Not sure of other capitalization at this level.